frr_python/frr_server.py

```diff
@@ -0,0 +1,72 @@
+"""A Fleetspeak server service
```

Comment:
FRR fleetspeak server

The first line should describe the relevant part of the file in one sentence.